frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/licenses/LicensesPortlet.java

```diff
@@ -92,14 +95,38 @@ public void doView(RenderRequest request, RenderResponse response) throws IOExce
             if (id != null) {
                 try {
                     LicenseService.Iface client = thriftClients.makeLicenseClient();
-                    License license = client.getByID(id, user.getDepartment());
-                    request.setAttribute(KEY_LICENSE_DETAIL, license);
+                    License moderationLicense = client.getByIDWithOwnModerationRequests(id, user.getDepartment(), user);
+
+                    List<Todo> addedTodos = moderationLicense.getTodos();
```

Comment:
It might be better to write
```Java
[...]
List<Todo> allTodos = nullToEmptyList(moderationLicense.getTodos());
List<Todo> addedTodos = addedTodos
        .stream()
        .filter(todo -> todo.id.startsWith("tmp"))
        .collect(Collectors.toList());
List<Todo>  currentTodos =  currentTodos
        .stream()
        .filter(todo -> !todo.id.startsWith("tmp"))
        .collect(Collectors.toList());

request.setAttribute(ADDED_TODOS_FROM_MODERATION_REQUEST, addedTodos);
request.setAttribute(DB_TODOS_FROM_MODERATION_REQUEST,currentTodos);
[...]
```